### PR TITLE
Update BalanceSheet for new probe prices method

### PR DESCRIPTION
**Pull request by haydenshively**
Updated the probe prices computation. Empirically the rest of the BalanceSheet still seems to work, but probably should do more testing once we have larger amounts deposited.

### CODE REVIEW

File: earn/src/data/BalanceSheet.ts [hunk 1]
```diff
@@ -3,54 +3,29 @@ import Big from 'big.js';
...
-    reward += (shortfall * price) / ALOE_II_LIQUIDATION_INCENTIVE;
-  }
-  if (liabilities1 > assets1) {
-    const shortfall = liabilities1 - assets1;
-    reward += shortfall / ALOE_II_LIQUIDATION_INCENTIVE;
-  }
-  return reward;
+  return [
+    prices.a.gt(MIN_SQRT_RATIO) ? prices.a : MIN_SQRT_RATIO,
+    prices.b.lt(MAX_SQRT_RATIO) ? prices.b : MAX_SQRT_RATIO,
```
Comment (IanWoodard) [inline earn/src/data/BalanceSheet.ts:27]: We could use GN.max and GN.min here

File: prime/src/data/BalanceSheet.ts [hunk 2]
```diff
@@ -3,53 +3,28 @@ import { GN } from 'shared/lib/data/GoodNumber';
...
+const PROBE_SQRT_SCALER_MAX = 3.078745359035;
 
-  sigma = sigma.recklessMul(nSigma);
-
-  let a = sqrtMeanPriceX96.mul(ONE.sub(sigma).sqrt());
-  let b = sqrtMeanPriceX96.mul(ONE.add(sigma).sqrt());
-
-  if (a.lt(MIN_SQRT_RATIO)) a = MIN_SQRT_RATIO.recklessAdd(1);
-  if (b.gt(MAX_SQRT_RATIO)) b = MAX_SQRT_RATIO.recklessSub(1);
+function _computeProbePrices(sqrtMeanPriceX96: GN, iv: GN, nSigma: number): [GN, GN] {
```
Comment (IanWoodard) [inline prime/src/data/BalanceSheet.ts:15]: Why are we still using different logic than in earn? Shouldn't we make them the same?

**Review by IanWoodard — APPROVED** [review]

File: earn/src/data/BalanceSheet.ts (hunk 1 above)
Comment (haydenshively) [reply earn/src/data/BalanceSheet.ts:27]: These are Bigs, not GNs

File: prime/src/data/BalanceSheet.ts (hunk 2 above)
Comment (haydenshively) [reply prime/src/data/BalanceSheet.ts:15]: I know it's kinda annoying, but I didn't think it was worth the time right now.

**Pull request merged**